src/browser_launchers/puppeteer_launcher.js

```diff
@@ -156,7 +158,8 @@ export class PuppeteerLauncher extends BrowserLauncher {
  * or the [Puppeteer proxy Example](../examples/puppeteer-with-proxy)
  *
  * @param {PuppeteerLaunchContext} [launchContext]
-* All `PuppeteerLauncher` parameters are passed via an launchContext object.
+ * All `PuppeteerLauncher` parameters are passed via an launchContext object.
+ * If you want to pass a custom `puppeteer.launch(options)` options you can use the `PuppeteerLaunchContext.launchOptions` property.
```

Comment:
`pass custom` without `a` because you're talking about option**S** which is plural :)